eth2near/contract_wrapper/src/eth_client_contract.rs

```diff
@@ -259,22 +259,12 @@ mod tests {
 
         let worker = rt.block_on(workspaces::sandbox()).unwrap();
         let wasm = std::fs::read(WASM_FILEPATH).unwrap();
-        let contract = rt.block_on(worker.dev_deploy(&wasm)).unwrap();
 
         // create accounts
         let owner = worker.root_account().unwrap();
-        let relay_account = rt
-            .block_on(
-                owner
-                    .create_subaccount("relay_account")
-                    .initial_balance(30 * near_sdk::ONE_NEAR)
-                    .transact(),
-            )
-            .unwrap()
-            .into_result()
-            .unwrap();
+        let contract = rt.block_on(owner.deploy(&wasm)).unwrap().unwrap();
 
-        (relay_account, contract)
+        (owner, contract)
```

Comment:
I think we need to return `(owner, relay_account, contract)`. For tests, it's especially mandatory - to verify the behavior of different accounts. The `owner` account should only be used on the initialization and in some negative tests.

Reply:
fixed: https://github.com/aurora-is-near/rainbow-bridge/pull/818/commits/4e20075373f66594873fdb5a86b0ad67dadaa62a